app/styles/components/_footer.scss

```diff
@@ -16,19 +16,41 @@ $icon-size: 25px;
   background-color: $Theme-color-blue;
 }
 
-.Footer-content {
-  @include font-properties(sans, desktop, content, large);
+.Footer-links {
+  @include font-properties(sans, desktop, content, small);
+  
+  display: flex;
+  flex-direction: column;
+}
 
-  margin-top: $Theme-margin-small;
+.Footer-link {
 
+  &:not(:first-child) {
+    margin-top: $Theme-margin-xSmall;
+  }
+
+  text-transform: uppercase;
   color: $Theme-color-white;
   font-weight: bold;
+
+  &:hover {
+    opacity: $Theme-opacity-half;
+  }
+}
+
+.Footer-content {
+  @include font-properties(sans, desktop, content, small);
+  display: flex;
+  flex-direction: column;
+  padding-top: $Theme-margin-small;
+
+  color: $Theme-color-white;
 }
 
 .Footer-column{
   display: flex;
   flex-direction: column;
-  align-self: flex-end;
+  align-self: flex-start;
```

Comment:
Why did this value change all of a sudden? Is it a consequence of removing the `text-align: left` property in `.Footer-credits`?